ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json

```diff
@@ -382,6 +382,25 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "judicialDecision",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-solicitor",
```

Comment:
the solicitor permissions will soon be removed once the JUDGE role assignments are done